docs/how_to_guides/example_models.rst

```diff
@@ -0,0 +1,133 @@
+oneself with **respy**'s functionalities and can be encountered in the tutorials.
+
+.. raw:: html
+
+     <div
+      <p class="d-flex flex-row gs-torefguide">
+          <span class="badge badge-info">To Tutorials</span></p>
+      <p>Check out the <a
+         href="../tutorials/index.html">Tutorials</a> </p>
+     </div>
+
+
+The models are called:
+
+- ``robinson_crusoe_basic``
+- ``robinson_crusoe_extended``
+
+These models are excellent examples to use for learning and prototyping because they are
+very computationally feasible due to the small number of available choices and low
+number of periods in the models.
+
+
+Keane and Wolpin (1994)
+-----------------------
+
+Aside from toy models, **respy** also provides several models that stem from the
+economic literature on dynamic life-cycle models. The most simple examples are a group
+of models based on the following publication:
+
+- Keane, M. P., & Wolpin, K. I. (1994). The Solution and Estimation of Discrete Choice
+  Dynamic Programming Models by Simulation and Interpolation: Monte Carlo Evidence.
+  *The Review of Economics and Statistics*, 648-672.
+
+
+The study is mainly concerned with the computational difficulties that accompany the
+solution of discrete choice dynamic programming (DCDP) problems such as the ones
+encountered in dynamic life-cycle models of educational and occupational choice. In the
+study, Keane and Wolpin (1994) develop an approximate solution method which consists of
+Monte Carlo integration with simulation and an interpolation approach to ease the
+computational burden of solving the DCDP model. They utilize one model with three
+different parametrizations to assess their solution method. This model and its three
+parametrizations are example models that are part of the **respy** interface.
+They are called:
+
+- ``kw_94_one``
+- ``kw_94_two``
+- ``kw_94_three``
+
+
+The model consists of four mutually exclusive alternatives that individuals can choose
+in each period. Agents can either choose to work in one of two sectors *a* or *b*,
+invest in *education* or stay *home*. The work alternatives award a wage and experience,
+while school only awards experience. In the home option, individuals gain neither a wage
+nor experience. The plot blow shows the choice patterns for the three parametrizations.
+The model considers a time horizon of 40 periods.
+
+
+Keane and Wolpin (1997)
+-----------------------
+
+A more advanced group of examples are given by the models developed by Keane and Wolpin
+(1997). In this study, the authors implement an empirical structural life-cycle model of
+occupational choice and human capital investment. They estimate their models on data
+from the National Longitudinal Survey of Youth (NLSY).
+
+- Keane, M. P., & Wolpin, K. I. (1997). The Career Decisions of Young Men.
+  *Journal of Political Economy*, 105(3), 473-522.
+
+
+**respy** supports both the basic and extended model from the paper. They are named:
```

Comment:
Maybe it is a little bit confusing to speak about the "basic and extended model", given the previous exposition. Probably better: basic and extended parametrization

Reply:
Good point, I also added some more text. :)